generate.py

```diff
@@ -49,7 +49,12 @@ def obj_to_cli(o, versions):
         html_file = "_dist/{}".format(out_file_names)
         json_file = "{}".format(out_file_names)
 
-        base = "python3 crashes.py -n {} -d {} -u {} -q {}".format(html_file, json_file, "https://sql.telemetry.mozilla.org", o["redash"])
+        crash_json_param = ""
+        if o["process_name"] not in ["main", "content"]:
```

Comment:
d owe really want to manually exclude those in the code this way? if we ever want to keep filtering this way (which I'm not sure, we should base our decision only on the lack of crash reports and the volumetry of the bug), I'd prefer if it was a CLI argument

Reply:
Good idea. It doesn't really hurt to generate them anyway, just a bit of extra processing (but realistically that doesn't take much time compared to all the symbolication).